dataminer/Functions/Dashboards_and_Low_Code_Apps/Low_Code_Apps/Style_guide.md

```diff
@@ -4,288 +4,210 @@ uid: LCA_Style_guide
+
+> [!NOTE]
+> If you come across something that has not been added to this style guide yet, [create a pull request](xref:CTB_Quick_Edit) to update this documentation.
 
 > [!TIP]
-> See also:
+> Are you new to designing low-code apps and dashboards? This will help you get started:
 >
 > - [Kata #19: Transform low-code apps into visual delights](https://community.dataminer.services/courses/kata-19/) on DataMiner Dojo ![Video](~/dataminer/images/video_Duo.png)
 > - Tutorial: [Creating a visually appealing and user-friendly low-code app](xref:Tutorial_App_Design)
 
-> [!NOTE]
-> If you come across something that has not been added to this style guide yet:
->
-> 1. Investigate what the best possible solution would be, ensuring consistency across the app and across the different available solutions.
-> 1. If you have any doubt, check with <rens.ampe@skyline.be>.
-> 1. [Create a pull request](xref:CTB_Quick_Edit) to update this documentation.
+## Themes
 
-## Colors/themes
+- When you choose the main color for a low-code app, ensure that there is enough contrast between the header, sidebar, and buttons.
 
-- When you create a new app or dashboard, it essential to start from a good **theme**. The same themes exist across the Dashboards and Low-Code Apps module within a DMS. The themes are primarily managed in the Dashboards app. If the theme you want to use does not exist yet, start by **duplicating a theme in the Dashboards app** that has a similar look and feel to what you have in mind, and then fine-tune it. In a low-code app, you can then select this theme on page level. Use the same theme for every page.
+- When you create a new app or dashboard, always start from a default theme. From DataMiner 10.4.0 [CU13]/10.5.0 [CU1]/10.5.4 onwards<!--RN 42179-->, the following default themes are available:
 
-  > [!TIP]
-  > See also: [Creating a new dashboard theme](xref:Configuring_the_dashboard_layout#creating-a-new-dashboard-theme)
+  - Skyline Light - White
 
-- Each theme **must** have a component style with a transparent background. Use this component style for titles, images, and buttons.
+  - Skyline Dark
 
-  ![Transparent theme](~/dataminer/images/LCA_Style_Guide_Theme_Transparent.png)
+- Use the same theme consistently across all pages and panels.
 
-- When you define a background color for pages/panels (as part of the theme), never use full white (#FFFFFF) or full black (#000000). Here are some good default colors:
+- [Customize component themes](xref:Customize_Component_Layout) only to make minor adjustments, such as removing padding inside a maps component.
 
-   |☀️  Light background |🌙 Dark background|
-   |---------------------|----------------|
-   |#F4F4F5              |#1E2530         |
+- Always use the *Default* component style, except for the following components, where you should use the *Transparent* style:
```

Comment:
Could you illustrate why we recommend this? If people understand the reason for this rule, they're more likely to follow it.